File: Print arrays in order/print_arrays_in_order.c
#include<stdio.h> 
void sort(int arr1[], int n) 
{ 
   int i, j,temp; 
   for (i = 0; i < n-1; i++)
    {   
       for (j = 0; j < n-i-1; j++)
       {
           if (arr1[j] > arr1[j+1])
           {
               temp = arr1[j]; 
               arr1[j] = arr1[j+1]; 
               arr1[j+1] = temp;
           }
       }
    }   
    return ;
} 
void rearrangeArray(int arr1[], int n) 
{   
    sort(arr1, n); 
    int tempArr[n];  
    int ArrIndex = 0;
    for (int i = 0, j = n-1; i <= n / 2 || j > n / 2; i++, j--) 
    { 
        tempArr[ArrIndex] = arr1[i]; 
        ArrIndex++; 
        tempArr[ArrIndex] = arr1[j]; 
        ArrIndex++; 
    } 
    for (int i = 0; i < n; i++) 
        {arr1[i] = tempArr[i];}
} 
int main() 
{ 
    int l;
    scanf("%d",&l);
    int arr1[l];  
    for(int i = 0; i < l; i++)
    {
        scanf("%d  ",&arr1[i]);
    }
    int n = sizeof(arr1) / sizeof(arr1[0]);  
    rearrangeArray(arr1, n); 
    for (int i = 0; i < n; i++) 
        printf("%d ",arr1[i]); 
    return 0; 
}

File: Print arrays in order/print_arrays_in_order.java
import java.util.Arrays; 
import java.util.Scanner;
public class GFG 
{  
    static void rearrangeArray(int arr[], int n) 
    {    
        Arrays.sort(arr); 
       
        int[] tempArr = new int[n];
        int ArrIndex = 0;
        for (int i = 0, j = n-1; i <= n / 2 || j > n / 2; 
                                           i++, j--) { 
            if(ArrIndex < n) 
            { 
                tempArr[ArrIndex] = arr[i]; 
                ArrIndex++; 
            } 
              
            if(ArrIndex < n) 
            { 
                tempArr[ArrIndex] = arr[j]; 
                ArrIndex++; 
            } 
        } 
        for (int i = 0; i < n; i++) 
            arr[i] = tempArr[i]; 
    } 
    public static void main(String args[]) 
    { 
        Scanner myObj = new Scanner(System.in);
        int n = myObj.nextInt();
        int arr[] =new int[n];
        for (int i = 0; i < n; i++)
        {
            arr[i] = myObj.nextInt(); 
        }
        rearrangeArray(arr, n); 
        
        for (int i = 0; i < n; i++) 
            System.out.print(arr[i]+" "); 
    } 
} 
